pageserver/src/config.rs

```diff
@@ -951,6 +978,9 @@ metric_collection_interval = '222 s'
 cached_metric_collection_interval = '22200 s'
 metric_collection_endpoint = 'http://localhost:80/metrics'
 synthetic_size_calculation_interval = '333 s'
+
+evictions_low_residence_duration_metric_threshold = '444 s'
```

Comment:
unsure what test got fixed with 44s -> 444s, but I had not been checking allure reports. Could you be more specific or comment here?

Reply:
The `parse_basic_config` test. Can we close this conversation?

Reply:
Aah oki so one more `4` was hit to the rhs of the `assert_eq!` in `parse_basic_config` and that caused one more `4` being needed here. Makes sense, I thought it was an actual test failure. Resolving.